buildscripts/semantic-convention/generate.sh

```diff
@@ -3,6 +3,8 @@
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 ROOT_DIR="${SCRIPT_DIR}/../../"
 # freeze the spec version to make SemanticAttributes generation reproducible
+# this hash was obtained by calling `git rev-parse master`
```

Comment:
can we not lock this to the recent 0.7.0 release for now?

Reply:
And if not, please reference the date that this version was pulled from.

Reply:
Added the date.